task2/config/webpack.common.js

```diff
@@ -0,0 +1,86 @@
+const util = require('./util');
+const {
+    CleanWebpackPlugin
+} = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const filename = (ext, subfolder = '') => `${subfolder}[name].${ext}`
+
+const jsLoaders = extra => {
+    const loaders = [{
+            loader: 'babel-loader',
+            options: babelOptions()
```

Comment:
I suppose because of `.babelrc` file no need to specify `options` here.

Reply:
Right! Thank you!